content/docs/csidriver/installation/helm/isilon.md

```diff
@@ -104,10 +104,10 @@ CRDs should be configured during replication prepare stage with repctl as descri
 1. Run `git clone -b v2.7.0 https://github.com/dell/csi-powerscale.git` to clone the git repository.
 2. Ensure that you have created the namespace where you want to install the driver. You can run `kubectl create namespace isilon` to create a new one. The use of "isilon"  as the namespace is just an example. You can choose any name for the namespace.
 3. Collect information from the PowerScale Systems like IP address, IsiPath, username, and password. Make a note of the value for these parameters as they must be entered in the *secret.yaml*.
-4. Copy *the helm/csi-isilon/values.yaml* into a new location with name say *my-isilon-settings.yaml*, to customize settings for installation.
+4. Download  `wget -O my-isilon-settings.yaml https://raw.githubusercontent.com/dell/helm-charts/main/charts/csi-isilon/values.yaml` into  `cd ../dell-csi-helm-installer` with name say *my-isilon-settings.yaml* , to customize settings for installation.
```

Comment:
why do we need to change the name of `values.yaml` file to `my-isilon-settings.yaml`, we can just use it as is?

Reply:
@chimanjain this is fine.
@HarishH-DELL pls remove  "with name say *my-isilon-settings.yaml*"